evidence.go

```diff
@@ -180,7 +182,7 @@ func (e *CcaEvidence) decodeClaims() error {
 	return nil
 }
 
-func (e *CcaEvidence) Verify(iak crypto.PublicKey, rpk crypto.PublicKey) error {
+func (e *Evidence) Verify(iak crypto.PublicKey) error {
```

Comment:
May be more relevant is the comment, here as to what Verify API has to offer.
Given that we have made it asymmetric to Sign

Reply:
I am not sure I fully understand the comment.  I think the gist is "write a doc comment"?

Reply:
I meant a simple statement stating that 
// Verify verifies the Platform token signature using supplied IAK and verifies Realm Token signature using inband key extracted from Realm Claims

Reply:
I got it right then :)

Reply:
done in [72c217d](https://github.com/veraison/ccatoken/pull/10/commits/72c217d6f261b82600cb4535ffcc6479fd50868f)